metrics/metrics.go

```diff
@@ -169,3 +177,14 @@ func CollectProcessMetrics(refresh time.Duration) {
 		time.Sleep(refresh)
 	}
 }
+
+func verifyPath(path string) (string, error) {
```

Comment:
repeated code i think

Reply:
Thanks!
Fixed in [dc41731e96](https://github.com/maticnetwork/bor/pull/578/commits/dc41731e96f110979832c6abb49c17ca813c54ff)